### ISSUE TITLE
Minor UI tweaks for PII request prevention

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli#5088]
- [ ] Could do with a touch more spacing between radio button and label
- [ ] Should make the font-size the same between the question and the labels

![screenshot 2019-01-31 at 09 03 42](https://user-images.githubusercontent.com/282788/52043260-2672d680-2537-11e9-81a8-2b7d37ae43d4.png)

**Comment by garethrees** [comment]
These are rendered in https://github.com/mysociety/alaveteli/blob/0.40.1.0/app/views/request/new.html.erb#L241-L244

**Comment by garethrees** [comment]
> Should make the font-size the same between the question and the labels

Open to options here – perhaps we _should_ emphasise the questions over the label? In any case, right now it just looks a bit broken rather than intentional.

**Comment by mdeuk** [comment]
> > Should make the font-size the same between the question and the labels
> 
> Open to options here – perhaps we _should_ emphasise the questions over the label? In any case, right now it just looks a bit broken rather than intentional.

The heading was improved in https://github.com/mysociety/alaveteli/commit/357a92a2eefe66cdfb7c1c4f7dfcd3a476331286 (#6800), so from a WDTK perspective it now looks like:

<img src="https://user-images.githubusercontent.com/249418/162419268-771fe582-e262-465c-8842-db198ac39123.png" width="500">

and, from my clunky sandbox, it looks like this in practice:
<img src="https://user-images.githubusercontent.com/249418/162423151-26213eef-4631-478b-88de-b0aaea866462.png" width="500">

Ideally, I think the text wrapping could be improved, so that it''s obvious which radio button relates to which item (the last one on the screenshot doesn't look so good!) - but otherwise, I think it looks reasonably okay.

**Comment by lucascumsille** [comment]
> > Should make the font-size the same between the question and the labels
> 
> Open to options here – perhaps we _should_ emphasise the questions over the label? In any case, right now it just looks a bit broken rather than intentional.

@garethrees Can you elaborate on the purpose of "are you asking about your's....." in terms of context? How is this question related to the ones below it?

This one doesn't have a radio button. Is it trying to remind the user to specify on behalf of who they are requesting the information in their request?

If that's the case, I'm tempted to suggest that we swap the order and let the radio button question goes first, and then we add this question. It seems that it is more of a remainder to provide that information than, unlikely the others, to specify the nature of the request, which seems more important.

**Comment by garethrees** [comment]
> Can you elaborate on the purpose of "are you asking about your's....." in terms of context? How is this question related to the ones below it?

Ah, so the problem we get on some authorities is users asking _inappropriate_ questions that are not FOI.

In the initial example, if a user is "asking about their visa", we turn them away from WDTK:

![Screenshot 2022-04-08 at 12 24 51](https://user-images.githubusercontent.com/282788/162426533-b853e0da-4220-47e8-839f-8d468086f3b8.png)

> This one doesn't have a radio button

I hadn't connected that some of this issue has been resolved in a different way via 357a92a2eefe66cdfb7c1c4f7dfcd3a476331286, where we've used a heading. Sorry for the confusion here.

I think the only remaining thing to do here is see if we can improve the text wrapping as @mdeuk points out:

>  I think the text wrapping could be improved, so that it''s obvious which radio button relates to which item (the last one on the screenshot doesn't look so good!)

So instead of this:

![Screenshot 2022-04-08 at 12 29 16](https://user-images.githubusercontent.com/282788/162427099-4d5d31ed-a42d-494d-9847-452c5d38f078.png)

We get this (red line just for emphasis of the change):

![Screenshot 2022-04-08 at 12 29 16](https://user-images.githubusercontent.com/282788/162427316-1f2621b0-6fa3-430c-88c4-896b8e6196d4.png)

**Comment by lucascumsille** [comment]
> > Can you elaborate on the purpose of "are you asking about your's....." in terms of context? How is this question related to the ones below it?
> 
> Ah, so the problem we get on some authorities is users asking _inappropriate_ questions that are not FOI.
> 
> In the initial example, if a user is "asking about their visa", we turn them away from WDTK:
> 
> ![Screenshot 2022-04-08 at 12 24 51](https://user-images.githubusercontent.com/282788/162426533-b853e0da-4220-47e8-839f-8d468086f3b8.png)
> 
> > This one doesn't have a radio button
> 
> I hadn't connected that some of this issue has been resolved in a different way via [357a92a](https://github.com/mysociety/alaveteli/commit/357a92a2eefe66cdfb7c1c4f7dfcd3a476331286), where we've used a heading. Sorry for the confusion here.
> 
> I think the only remaining thing to do here is see if we can improve the text wrapping as @mdeuk points out:
> 
> > I think the text wrapping could be improved, so that it''s obvious which radio button relates to which item (the last one on the screenshot doesn't look so good!)
> 
> So instead of this:
> 
> ![Screenshot 2022-04-08 at 12 29 16](https://user-images.githubusercontent.com/282788/162427099-4d5d31ed-a42d-494d-9847-452c5d38f078.png)
> 
> We get this (red line just for emphasis of the change):
> 
> ![Screenshot 2022-04-08 at 12 29 16](https://user-images.githubusercontent.com/282788/162427316-1f2621b0-6fa3-430c-88c4-896b8e6196d4.png)

Great thanks for clarifying that, I was working atm on the latest issue(last screenshot). Also I saw the line-height is quite big when is a two line question.

**Comment by garethrees** [comment]
> Also I saw the line-height is quite big when is a two line question.

+1, good spot and an improvement there is worth making while you're working on this.
